pkgs/applications/networking/ipget/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchgx }:
+
+buildGoPackage rec {
+  name = "ipget-${version}";
+  version = "0.2.5";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/ipfs/ipget";
+  
+  extraSrcPaths = [
```

Comment:
what's the relation between `fetchgx` and `fetchFromGitHub` sources?

Reply:
There is no relation. As I understand, ```fetchgx``` learns about relevant dependencies independently via IPNS and then downloads everything from IPFS. On our side we just put all dependencies in one fixed-hash derivation. I think this is also the same way rust/cargo in nixpkgs currently work in nixpkgs.